packages/dependency-extraction-webpack-plugin/lib/util.js

```diff
@@ -64,6 +72,15 @@ function defaultRequestToHandle( request ) {
 
 		case 'lodash-es':
 			return 'lodash';
+
+		// Ignore object-assign ponyfill, which is used by jsx-(dev)-runtime.
+		case 'object-assign':
```

Comment:
Can you clarify more this addition?

Reply:
React internally is using `object-assign` as a ponyfill since that they have to support IE. Unfortunately, it's increasing the bundle size and we don't need to support IE anymore. The setting here maps `var assign = require('object-assign')` to `var assign = window['Object']['assign']` to avoid bundling the package.

Reply:
Makes sense thanks.